flume/doc/operation/hdfs_kerberos_authentication.md

```diff
@@ -0,0 +1,67 @@
+# Enabling Kerberos SPNEGO authentication in HDFS
+
+Hadoop Distributed File System (HDFS) can be remotely managed through a REST API called [WebHDFS](http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/WebHDFS.html). This API may be used without any kind of security (in this case, it is enough knowing a valid HDFS user name in order to access this user HDFS space), or a Kerberos infrastructure may be used for authenticating the users.
+
+Kerberos is an authentication protocol created by MIT, current version is 5. It is based in symmetric key cryptography and a trusted third party, the Kerberos servers themselves. The protocol is as easy as authenticating to the Authentication Server (AS), which forwards the user to the Key Distribution Center (KDC) in order to create a ticket-granting ticket (TGT). This ticket can then be used for authentication purposes against a service server; this service server takes the TGT and verifies it is valid against the KDC.
+
+SPNEGO is a mechanism used to negotiate the choice of security technology. Through SPNEGO both client and server may negotiate the usageof Kerberos as authentication technology.   
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/af19dc77924e4d15f2d0dfc86ac8d0ce2cf30c63